public/locales/uk/translation.json

```diff
@@ -26,6 +26,10 @@
     "DESCRIPTION": "Ви можете знайти проект, який підходить вам ідеально, і дізнатися, як ваша компанія може підтримати його.",
     "HEADING": "Соціальні та благодійні ініціативи України"
   },
+  "about": {
```

Comment:
please keep in alphabetical order. "about" should go before "footer"

Reply:
Done!